src/_data/sidenav.yml

```diff
@@ -314,6 +314,8 @@
     # - title: Flutter learning resources # TODO
     - title: FAQ
       permalink: /docs/resources/faq
+    - title: Design Documents
+      permalink: /docs/resources/design-docs
```

Comment:
@sfshaza2  - this is what you were asking for right?

It looks like it's working in your latest staging of this.